Userland/Libraries/LibC/time.cpp

```diff
@@ -275,6 +276,19 @@ char* asctime_r(const struct tm* tm, char* buffer)
     return buffer;
 }
 
+char* strptime(char const* input, char const* format, struct tm* tm)
+{
+    size_t format_length = strlen(format);
+    size_t input_length = strlen(format);
+    size_t string_pos = 0;
+    auto tm_or_void = AK::convert_formatted_string_to_timespec(StringView(input, input_length), StringView(format, format_length), string_pos);
```

Comment:
I'm not sure if there is any actual difference, but we usually do the following instead:

```suggestion
    auto tm_or_void = AK::convert_formatted_string_to_timespec(StringView { input, input_length }, StringView { format, format_length }, string_pos);
```

That said, it might be worth it readability wise to do `StringView format_view { format, strlen(format) };` (`input` respectively) as the very first thing in the function and then just use `.length()` everywhere.